### ISSUE TITLE
Receiver Controls not available

### ISSUE THREAD

**Issue by lfloeer** [ACDR/homebridge-yamaha-avr#34]
Edit: Forgot to say thanks for this great plugin! Thank you 😅

**Describe The Bug:**
When adding my Yamaha RX-V485 I only get the "Speaker" accessory, but no input or TV controls in Home. There is a log message about adding a duplicate input in the accessory but I can't see a way of fixing that myself.

**Logs:**

```
[10/21/2020, 8:41:47 AM] [HB Supervisor] Sending SIGTERM to Homebridge
[10/21/2020, 8:41:47 AM] Got SIGTERM, shutting down Homebridge...
[10/21/2020, 8:41:52 AM] [HB Supervisor] Homebridge Process Ended. Code: 143, Signal: null
[10/21/2020, 8:41:57 AM] [HB Supervisor] Restarting Homebridge...
[10/21/2020, 8:41:57 AM] [HB Supervisor] Starting Homebridge with extra flags: -I -P /homebridge/node_modules
[10/21/2020, 8:41:57 AM] [HB Supervisor] Started Homebridge v1.1.6 with PID: 985
[10/21/2020, 8:41:59 AM] Loaded config.json with 4 accessories and 2 platforms.
[10/21/2020, 8:41:59 AM] ---
[10/21/2020, 8:42:03 AM] Loaded plugin: homebridge-mqttthing@1.1.20
[10/21/2020, 8:42:03 AM] Registering accessory 'homebridge-mqttthing.mqttthing'
[10/21/2020, 8:42:03 AM] ---
[10/21/2020, 8:42:04 AM] Loaded plugin: homebridge-yamaha-avr@2.0.2
[10/21/2020, 8:42:04 AM] Registering platform 'homebridge-yamaha-avr.yamaha-avr'
[10/21/2020, 8:42:04 AM] ---
[10/21/2020, 8:42:04 AM] Loaded plugin: homebridge-config-ui-x@4.29.0
[10/21/2020, 8:42:04 AM] Registering platform 'homebridge-config-ui-x.config'
[10/21/2020, 8:42:04 AM] ---
[10/21/2020, 8:42:04 AM] Loading 2 platforms...
[10/21/2020, 8:42:04 AM] [Config] Initializing config platform...
[10/21/2020, 8:42:04 AM] [Config] Running in Service Mode
[10/21/2020, 8:42:04 AM] [Receiver] Initializing yamaha-avr platform...
[10/21/2020, 8:42:04 AM] Loading 4 accessories...

... a few lines later ...

(node:985) UnhandledPromiseRejectionWarning: Error: Cannot add a Service with the same UUID '000000D9-0000-1000-8000-0026BB765291' and subtype 'AV2' as another Service in this Accessory.
    at Accessory._this.addService (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:281:17)
    at PlatformAccessory.addService (/usr/local/lib/node_modules/homebridge/src/platformAccessory.ts:92:41)
    at /homebridge/node_modules/homebridge-yamaha-avr/src/accessory.ts:229:45
(node:985) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:985) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Plugin Config:**

```json
        {
            "ip": "192.168.1.3",
            "name": "Receiver",
            "cacheDirectory": "/homebridge/yamaha-avr-persist",
            "platform": "yamaha-avr"
        }
```

**Environment:**

* **Plugin Version**: 2.0.2
* **Homebridge Version**: 1.1.6
* **Node.js Version**: v12.18.3
* **NPM Version**: 6.14.6
* **Operating System**: Docker Image oznu/homebridge:raspberry-pi

**Comment by ACDR** [comment]
Strange - looks like your AVR is providing two "AV2" inputs. I've published a new release, 2.0.3 to catch the error and ignore duplicate inputs.

The Speaker accessory is separate to this plugin/Homebridge - it's a 1st party AirPlay 2 Speaker accessory for models that support it.

**Comment by lfloeer** [comment]
That did the trick. Using version 2.0.4 of the plugin I can add the receiver without issue. Thank you!
